lib/v2/lsnu/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'jiaowc.lsnu.edu.cn': {
```

Comment:
Do not include subdomain in top level domain key.
Docs: https://docs.rsshub.app/joinus/new-radar